src/test/scala/org/broadinstitute/dsde/firecloud/service/NihServiceUnitSpec.scala

```diff
@@ -102,14 +118,18 @@ class NihServiceUnitSpec extends AnyFlatSpec with Matchers with BeforeAndAfterEa
     userNoAllowlists.id -> userNoAllowlistsLinkedAccount,
     userTcgaAndTarget.id -> userTcgaAndTargetLinkedAccount,
     userTcgaOnly.id -> userTcgaOnlyLinkedAccount,
-    userTargetOnly.id -> userTargetOnlyLinkedAccount
+    userTargetOnly.id -> userTargetOnlyLinkedAccount,
+    userDbGapList.id -> userDbGapListLinkedAccount,
+    userDbGapGroup.id -> userDbGapGroupLinkedAccount
   )
 
   val linkedAccountsByExternalId = Map(
     userNoAllowlistsLinkedAccount.linkedExternalId -> userNoAllowlistsLinkedAccount,
     userTcgaAndTargetLinkedAccount.linkedExternalId -> userTcgaAndTargetLinkedAccount,
     userTcgaOnlyLinkedAccount.linkedExternalId -> userTcgaOnlyLinkedAccount,
-    userTargetOnlyLinkedAccount.linkedExternalId -> userTargetOnlyLinkedAccount
+    userTargetOnlyLinkedAccount.linkedExternalId -> userTargetOnlyLinkedAccount,
+    userDbGapListLinkedAccount.linkedExternalId -> userDbGapListLinkedAccount,
+    userDbGapGroupLinkedAccount.linkedExternalId -> userDbGapGroupLinkedAccount
```

Comment:
Instead of changing existing tests, I left them without any results from the search by consentGroup + phsId, and instead created a new test where we get one user from that and another user from the allowlist file for a new allowlist.  Not sure what kind of situations we should expect to exist or to test for, so I'd love some suggestions as to what combinations I should set up here for testing.